server/go.mod

```diff
@@ -59,7 +59,9 @@ replace (
 
 // Bad imports are sometimes causing attempts to pull that code.
 // This makes the error more explicit.
-replace (
-	go.etcd.io/etcd => ./FORBIDDEN_DEPENDENCY
-	go.etcd.io/tests/v3 => ./FORBIDDEN_DEPENDENCY
-)
+replace go.etcd.io/etcd => ./FORBIDDEN_DEPENDENCY
+
+replace go.etcd.io/tests/v3 => ./FORBIDDEN_DEPENDENCY
+
+// Avoid dependency on go.etcd.io/etcd.
+replace github.com/go-kit/kit v0.10.0 => github.com/go-kit/kit v0.10.1-0.20210530035833-3b053a73c0f4
```

Comment:
Do we still need this? I thought it was solved by todays prometheus/common release? Or does it not come from client_golang?

Reply:
It comes from `prometheus/client_golang`
I might try to depend on https://github.com/prometheus/client_golang/commit/253906201bda760621fa671fa1541a4ac3df29bd instead.

Reply:
I see that was not released yet, makes sense. @kakkoyun is there any plans to maybe release a patch release? Thank you!

Reply:
Thank you for the suggestion. As in both scenarios we depend on unstable version, I found depending on ustable `prometheus/client_golang` cleaner. PTAL.

Reply:
No plans yet for a need patch release. I'd advise to use a master version :blush:

Reply:
It means that for release-3.5 we need to stick to last stable version that does not introduce a cyclic dependency, i.e: 
github.com/prometheus/client_golang v1.5.1 (https://github.com/etcd-io/etcd/blob/fea5ace182a74c29da64c8283ea0a84203c2d3f2/client/v3/go.mod#L8)

Reply:
Not sure how the k8s deps would play with that?